TechnicoBackend/Controllers/PropertyController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using TechnicoBackend.Services;
+using TechnicoBackend.Models;
+using TechnicoBackend.DTOs;
+
+namespace TechnicoBackend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class PropertyController : ControllerBase
+    {
+        private readonly PropertyService _propertyService;
```

Comment:
Consider initializing an IPropertyService here instead of PropertyService, which would be mapped accordingly using proper Dependency Injection in Program.cs.